src/Auth/Directives.js

```diff
@@ -108,11 +108,34 @@ class AuthenticatedDirective extends SchemaDirectiveVisitor {
   }
 }
 
+class OwnerDirective extends SchemaDirectiveVisitor {
 
+  visitObject(type){
+    this.wrapOwnerAuth(type);
+    type._requiresOwAuth = true;
+  }
+    
+  visitFieldDefinition(field) {
+    const { resolve = defaultFieldResolver } = field;
+    field.resolve = async function (record, args, context, info){
+
+      const OwnerRequester = await getOwnerid(context.token.owner);
+      const OwnerRequested = await getOwnerid(record);
+
+      if(OwnerRequester !== null && OwnerRequested !== null){
+        if(OwnerRequester == OwnerRequested){
+            return resolve.apply(this, [record, args, context, info]);
+        }
+      }
 
+      return await blockValue(field);
+    }
```

Comment:
Missing `;` on line 132 (eslint).  It's the end of a value assignment to `field.resolve`.